lib/utils.js

```diff
@@ -7,7 +7,7 @@ var camelCaseAttrMap = require('./camel-case-attribute-names');
 
 function createStyleJsonFromString(styleString) {
   styleString = styleString || '';
-  var styles = styleString.split(';');
+  var styles = styleString.split(/;(?!base64)/);
```

Comment:
This regex says to split on all `;` that don't `(?!` have the trailing string `base64)`